packages/components/src/components/select-native/select-native.jsx

```diff
@@ -29,6 +39,7 @@ const SelectNative = ({
     suffix_icon,
     use_text,
     value,
+    hide_disabled,
```

Comment:
```suggestion
    should_hide_disabled,
```

Reply:
Thanks @zaki-binary. changed to should_hide_disabled_options as @aaron-binary suggested